src/screens/TxDetails.js

```diff
@@ -96,9 +90,6 @@ export class TxDetailsView extends React.PureComponent {
           title={this.props.sender.name}
           address={this.props.sender.address}
           chainId={this.props.sender.chainId}
-          onPress={() => {
-            this.props.onPressAccount(this.props.sender);
```

Comment:
Has there been any issues with navigation here?

(I don't think I ever wanted to navigate from tx screen to account by tapping the icon, or expected that to work that way, so I think it's fine, just curious about the reasons)

Reply:
There has been no issue opened for this indeed. I stumbled upon this and would not expect this either, hence the removal. I have the bad habit to tackle several issues at once :). I did my best not to shoot in too many directions for this one.

Reply:
> I have the bad habit to tackle several issues at once :)

how sweet is it